server/index.js

```diff
@@ -11,24 +11,27 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.use(express.static(__dirname + '/../client/dist'));
 
 // categories
-// app.get('/categories', wrap(async (req, res) => {
-//   const categories = await db.Category.findAll();
-//   res.json(categories);
-// }));
+app.get('/categories', wrap(async (req, res) => {
+  const categories = await db.Category.findAll();
+  res.json(categories);
+}));
 
 // events
-// app.get('/events', wrap(async (req, res) => {
-//   const { categoryId } = req.query;
-//   let events = [];
+app.get('/events', wrap(async (req, res) => {
+  const { categoryId } = req.query;
+  const { Subcategories } = await db.Category.findById(categoryId, { include: [{ model: db.Subcategory, include: db.Event }] });
+  let events = [];
+  for (let i = 0; i < Subcategories.length; i++) {
+    events = events.concat(Subcategories[i].Events);
+  }
+  res.json(events);
+}));
 
-//   if (categoryId) {
-//     const category = await db.Category.findById(categoryId);
-//     events = await category.getEvents();
-//   } else {
-//     events = await db.Event.findAll();
-//   }
+// articles
+app.get('/articles', wrap(async (req, res) => {
```

Comment:
Where do we want to do the work of getting just the articles that the client cares about?